### PR TITLE
fix(designer): Html Editor not showing proper tokens

### PR DESCRIPTION
**Pull request by Eric-B-Wu**
Fixes -https://github.com/Azure/LogicAppsUX/issues/3497

Other HTML ui bugs fixed
- Text from HTML node (extendedTextNode) uncopyable
- Tokenpickerbutton not properly being offset in new lines of HTML editors
Other
- Adding a prop to also make it possible to only show the expression editor

### CODE REVIEW

**Review by hartra344 — APPROVED** [review]

File: libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx [hunk 1]
```diff
@@ -66,13 +67,14 @@ function TokenMenuItem({
 }
 
 interface TokenTypeAheadPluginProps {
-  openTokenPicker: (tokenPickerMode: TokenPickerMode) => void;
   isEditorFocused?: boolean;
-  hideExpression?: boolean;
+  hideTokenpickerOptions?: hideButtonOptions;
```
Comment (rllyy97) [inline libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx:71]: Just for consistent capitalization
```suggestion
  hideTokenPickerOptions?: hideButtonOptions;
```

**Review by rllyy97 — APPROVED** [review]

File: libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx (hunk 1 above)
Comment (rllyy97) [reply libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx:71]: rip nevermind I was too slow

Comment (Eric-B-Wu) [reply libs/designer-ui/src/lib/editor/base/plugins/TokenTypeahead.tsx:71]: oops, ill make this change in my next PR

**Comment by FleeceOfGideon** [comment]
Can I just check this got rolled out to all regions? (Sorry if stupid question but it is the 1st time following an issue I am impacted #3497)

**Comment by FleeceOfGideon** [comment]
@Eric-B-Wu , has this been released? If not, any ETA?

**Pull request merged**